### ISSUE TITLE
mget fails for large files over slow internet

### ISSUE THREAD

**Issue by notmatt** [joyent/node-manta#208]
Attempting to download a large (2GB) file over slow internet yielded this:
```
$ mget ~~/stor/big.tgz -O big.tgz
big.tgz [===========>                           ]  32% 765.19MB 476.60KB/s 55m51smget: RequestExpiredError: Date header Fri, 21 Nov 2014 01:45:31 GMT is too old
```
Where 01:45:31 is approximately the time of the original request. This occurs reliably for me, though at varying times after the initial request.

**Comment by bahamat** [comment]
Filed internally as MANTA-3546.

**Comment by lloydde** [comment]
https://smartos.org/bugview/MANTA-2546

**Comment by bahamat** [comment]
Fixed in commit 9327891.
